.github/workflows/conformance.yml

```diff
@@ -118,7 +120,7 @@ jobs:
         - component: pubsub.azure.servicebus
           required-secrets: AzureServiceBusConnectionString
         - component: bindings.azure.blobstorage
-          required-secrets: AzureBlobStorageAccessKey,AzureBlobStorageAccount,AzureBlobStorageContainer
```

Comment:
This didn't need to be a secret and could be hardcoded (for simplicity i.e. when running locally)